lib/shared/relative-module-resolver.js

```diff
@@ -11,8 +11,7 @@ const Module = require("module");
  * `Module.createRequire` is added in v12.2.0. It supports URL as well.
  * We only support the case where the argument is a filepath, not a URL.
  */
-// eslint-disable-next-line node/no-unsupported-features/node-builtins, node/no-deprecated-api
-const createRequire = Module.createRequire || Module.createRequireFromPath;
+const createRequire = Module.createRequire || Module.createRequireFromPath; // eslint-disable-line node/no-unsupported-features/node-builtins, node/no-deprecated-api
```

Comment:
Can we keep these as two separate lines? Easier to read :)

Reply:
as we are using tag `template`, see https://github.com/gajus/eslint-plugin-jsdoc#check-tag-names